Src/Plugins/Tests.Security/Helpers/HttpMockHelper.cs

```diff
@@ -101,6 +105,7 @@ public struct HttpMockTestCase
         public List<HttpContent> HttpContents { get; set; }
         public List<HttpStatusCode> HttpStatusCodes { get; set; }
         public List<HttpRequestMessage> HttpRequestMessages { get; set; }
+        public List<HttpResponseMessage> HttpResponseMessages { get; set; }
```

Comment:
>List [](http://example.com/codeflow?start=15&length=4)

Make these IList, generally use the abstraction for public things. It's just a good practice.

Reply:
thanks for reviewing.
just updated.